src/rpcgovernance.cpp

```diff
@@ -623,6 +623,18 @@ UniValue voteraw(const UniValue& params, bool fHelp)
 
 UniValue getgovernanceinfo(const UniValue& params, bool fHelp)
 {
+    int nLastSuperblock, nNextSuperblock;
+
+    // Get current block height
+    int nBlockHeight = (int)chainActive.Height();
```

Comment:
You should probably do a LOCK(cs_main) before making this call.

This call calls .size() on a vector other threads can modify.

See: http://stackoverflow.com/questions/2377712/on-macosx-using-g-is-stdvector-size-thread-safe